src/saving/FileHelpers.cs

```diff
@@ -43,4 +45,28 @@ public static bool Exists(string path)
         using var directory = new Directory();
         return directory.FileExists(path);
     }
+
+    /// <summary>
+    ///   Returns true if file exists and the file name matches case
+    /// </summary>
+    /// <param name="path">Path to check</param>
+    /// <returns>True if exists, false otherwise</returns>
+    public static bool ExistsCaseSensitive(string path)
+    {
+        // Checks if it exist first before checking case
+        if (!Exists(path))
+        {
+            return false;
+        }
+
+        var globlizedPath = ProjectSettings.GlobalizePath(path);
```

Comment:
I guess you intended to name the variable `globAlizedPath` instead (caps A for highlighting, of course)?

Reply:
yeah just misspelled